rand_core/src/lib.rs

```diff
@@ -76,12 +76,17 @@ pub mod le;
 /// [`next_u32`] or [`next_u64`] since the latter methods are almost always used
 /// with algorithmic generators (PRNGs), which are normally infallible.
 ///
+/// Implementers should produce bits uniformly. Pathological RNGs (e.g. always
+/// returning the same value, or never setting certain bits) can break rejection
+/// sampling used by random distributions, and also break other RNGs when
+/// seeding them via [`SeedableRng::from_rng`].
```

Comment:
What do you mean with breaking other RNGs?

Reply:
For example, xorshift failing to initialize if a pathological RNG returns only zeroes. Should I give that specific example in the doc?

Reply:
Fair enough! No, this is fine.